src/tiled_node/tiled_node.cpp

```diff
@@ -502,27 +542,27 @@ static inline Vector3i oddr_to_axial(Vector3i oddr) {
     return Vector3i(q, oddr.y, oddr.z);
 }
 
-HexMapCellId HexMap::local_to_cell_id(const Vector3 &local_position) const {
-    Vector3 unit_pos = local_position / get_cell_scale();
-    return HexMapCellId::from_unit_point(unit_pos);
+HexMapCellId HexMapTiledNode::local_to_cell_id(
+        const Vector3 &local_position) const {
+    return space.get_cell_id(local_position);
 }
 
-Ref<hex_bind::HexMapCellId> HexMap::_local_to_cell_id(
+Ref<hex_bind::HexMapCellId> HexMapTiledNode::_local_to_cell_id(
         const Vector3 &p_local_position) const {
     return local_to_cell_id(p_local_position);
 }
 
-Vector3 HexMap::cell_id_to_local(const HexMapCellId &cell_id) const {
-    return cell_id.unit_center() * get_cell_scale() + get_cell_mesh_offset();
+Vector3 HexMapTiledNode::cell_id_to_local(const HexMapCellId &cell_id) const {
+    return space.get_cell_center(cell_id);
 }
 
-Vector3 HexMap::_cell_id_to_local(
+Vector3 HexMapTiledNode::_cell_id_to_local(
         const Ref<hex_bind::HexMapCellId> p_cell_id) const {
     ERR_FAIL_COND_V_MSG(!p_cell_id.is_valid(), Vector3(), "null cell id");
     return cell_id_to_local(**p_cell_id);
 }
```

Comment:
HexMapNode has better implementations of these, so we should just leave them to it.  Remove these.